Quotient/events/serveraclevent.h

```diff
@@ -0,0 +1,19 @@
+// SPDX-FileCopyrightText: 2024 James Graham <james.h.graham@protonmail.com>
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#pragma once
+
+#include "stateevent.h"
+
+namespace Quotient {
+class QUOTIENT_API ServerAclEvent : public StateEvent {
+public:
+    QUO_EVENT(ServerAclEvent, "m.room.server_acl")
+
+    using StateEvent::StateEvent;
+
+    QStringList allow() const;
+    bool allowIpLiterals() const;
+    QStringList deny() const;
```

Comment:
```suggestion
    QUO_CONTENT_GETTER(QStringList, allow)
    QUO_CONTENT_GETTER(bool, allowIpLiterals)
    QUO_CONTENT_GETTER(QStringList, deny)
```
And you don't need the .cpp file at all; but if (see the headline comment) you keep this file, please add it to `CMakeLists.txt`.

Reply:
Happy to move into `simplestateevents.h` and get rid of the rest, didn't know this was an option.